src/components/DisplayNames/index.js

```diff
@@ -5,6 +5,9 @@ import {propTypes, defaultProps} from './displayNamesPropTypes';
 import styles from '../../styles/styles';
 import Tooltip from '../Tooltip';
 import Text from '../Text';
+import variables from '../../styles/variables';
+
+const TOOLTIP_MAX_LINES = 3;
```

Comment:
I meant in CONST.js file.

Reply:
Oh, my bad, I wasn't aware of that file. Fixing it.